src/game_initialization/connect_engine.cpp

```diff
@@ -1066,43 +1066,10 @@ config side_engine::new_config() const
 			(*leader)["gender"] = "null";
 		}
 
-		res["team_name"] = parent_.team_data_[team_].team_name;
+		std::string& new_team_name = parent_.team_data_[team_].team_name;
```

Comment:
Is there a reason this is not a const reference?

Reply:
fixed